Qlc/Objects/HistoricalRewardInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Qlc.Objects
+{
+    public class HistoricalRewardInfo
```

Comment:
change HistoricalRewardInfo to MinerHistoricalRewardInfo just for clear.

Reply:
Done